### DISCUSSION TITLE
Linkwarden LXC created as a privileged container

### DISCUSSION THREAD

**Discussion by TheRealAvise** [tteck/Proxmox#2266] [General]
Hi,

Thanks for all the great work, your scripts are a big help for beginners like me.

Anyway, I noticed that the Linkwarden LXC creation script creates the LXC as privileged if using default settings. There does not seem to be any need for that from what I understand.

**Comment by tteck** [comment]
True 🤷🏻‍♂️ https://github.com/tteck/Proxmox/commit/77cd36e094401fc6e2f0e5552fd5956a373fcf29
